integration-tests/vitest/vitest.spec.js

```diff
@@ -375,15 +375,15 @@ versions.forEach((version) => {
           const testSuite = events.find(event => event.type === 'test_suite_end').content
           assert.equal(test.meta[TEST_CODE_OWNERS], JSON.stringify(['@datadog-dd-trace-js']))
           assert.equal(testSuite.meta[TEST_CODE_OWNERS], JSON.stringify(['@datadog-dd-trace-js']))
-        })
+        }, 25000)
```

Comment:
unrelated change: this test is a bit slow so we need to wait for the data to arrive